test/measurable_test.rb

```diff
@@ -152,9 +152,9 @@ module Example
   end
 
   test "#<=> compares regardless of the unit" do
-    assert_equal -1, @magic <=> Magic.new(10, :fire)
+    assert_equal -1, @magic <=> Magic.new(20, :fire)
```

Comment:
This was actually wrong / a bug before, because we were doing `BigDecimal("2/3")` for `@magic`, which silently becomes 2...